definitions/npm/lodash_v4.x.x/flow_v0.104.x-/lodash_v4.x.x.js

```diff
@@ -275,7 +275,7 @@ declare module "lodash" {
     flatten<T, X>(array?: ?$ReadOnlyArray<$ReadOnlyArray<T> | X>): Array<T | X>;
     flattenDeep<T>(array?: ?(any[])): Array<T>;
     flattenDepth(array?: ?(any[]), depth?: ?number): any[];
-    fromPairs<A, B>(pairs?: ?Array<[A, B]>): { [key: A]: B, ... };
+    fromPairs<A, B>(pairs?: ?$ReadOnlyArray<[A, B]>): {| [key: A]: B |};
```

Comment:
The exact object makes more sense here. 

But I'm not sure whether it was implicit inexact to explicit inexact conversion or it was done on purpose in this PR https://github.com/flow-typed/flow-typed/pull/3467

@jbrown215 could you clarify, please?

Reply:
This was the result of an automated script, that could not distinguish all nuances.  
I think your changes makes sense.